popups.css

```diff
@@ -1,4 +1,5 @@
 .lsp_popup {
+    --mdpopups-hr: color(var(--foreground) alpha(0.10));
```

Comment:
While I don't know much of CSS, the `--mdpopups-hr` looks out of place. Why can't you style `hr` directly? I suppose that would influence the signature help as well?

Reply:
Changed it to style the hr directly.